tests/test_ankle_draft.py

```diff
@@ -1,13 +1,23 @@
 import pytest
-
 from pythermalcomfort.models import ankle_draft
 
+def test_ankle_draft(get_ankle_draft_url, retrieve_data):
+    test_data = retrieve_data(get_ankle_draft_url)
+
+    if test_data is None:
+        pytest.skip("Failed to retrieve test data")
 
-def test_ankle_draft():
-    assert (ankle_draft(25, 25, 0.2, 50, 1.2, 0.5, 0.3, units="SI")["PPD_ad"]) == 18.5
-    assert (
-        ankle_draft(77, 77, 0.2 * 3.28, 50, 1.2, 0.5, 0.4 * 3.28, units="IP")["PPD_ad"]
-    ) == 23.5
+    # Tests using data from URL
+    for case in test_data.get("data", []):
```

Comment:
the code should be the same across different test functions. in the test adaptive for example, you wrote the code

for entry in reference_table["data"]: while here you use the get function. I like this approach more `for entry in reference_table["data"]:` but we need to be consistent